### ISSUE TITLE
Restrict duplicate  patient import from hdp-adapter

### ISSUE THREAD

**Issue by andreakreichgauer** [dot-base/fhir-server#112]
### 🚀 Feature Description
As a user I want to import a patient from the hospital's primary system (via hdp-adapter) only once. This is check is already done by the hdp-adapter, but can be bypassed, if a patient-import is triggered multiple times within seconds.

Thus, the fhir server should enforce, that new patients have a unique [primary hospital identifier](https://github.com/dot-base/fhir-resources/blob/master/resources/dotbase/NamingSystem/PatientPrimaryIdentifier.json). If a Patient with the according identifier already exists, patient creation should fail.

### ✔ Goals
- [x] Add a interceptor that checks for duplicate ids before a Patient resource is created
